### PR TITLE
Add animation support for draco encoded FBXs

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3357/HighFidelity-Beta-PR11369-cc13fda4063c482053b05254ef8fe1e09a1e7f73.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3357/HighFidelity-Beta-PR11369-cc13fda4063c482053b05254ef8fe1e09a1e7f73.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3368/HighFidelity-Beta-PR11369-c5852dfbe81ca620c0bdf6c2b8178e91241ba3f2.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3368/HighFidelity-Beta-PR11369-c5852dfbe81ca620c0bdf6c2b8178e91241ba3f2.exe

File: libraries/fbx/src/FBXWriter.cpp [hunk 1]
```diff
@@ -13,14 +13,45 @@
...
+    if (data.size() >= ATTEMPT_COMPRESSION_THRESHOLD_BYTES) {
+        auto compressedDataWithLength { qCompress(data) };
+
+        // qCompress packs a length uint32 at the beginning of the buffer, but the FBX format
+        // does not expect it. This removes it.
+        auto compressedData = QByteArray::fromRawData(
+            compressedDataWithLength.constData() + sizeof(uint32_t), compressedDataWithLength.size() - sizeof(uint32_t));
+
+        if (compressedData.size() < data.size()) {
+            out << (int32_t)1;
```
Comment (Atlante45) [inline libraries/fbx/src/FBXWriter.cpp]: FBX_PROPERTY_COMPRESSED_FLAG

File: libraries/fbx/src/FBXWriter.cpp [hunk 2]
```diff
@@ -13,14 +13,45 @@
...
+            out << (int32_t)compressedData.size();
+            out.writeRawData(compressedData.constData(), compressedData.size());
+            return;
+        }
+    }
+
+    out << FBX_PROPERTY_UNCOMPRESSED_FLAG;
+    out << (int32_t)0;
```
Comment (Atlante45) [inline libraries/fbx/src/FBXWriter.cpp]: Aren't you supposed to pack the size here?

Comment (huffman) [reply libraries/fbx/src/FBXWriter.cpp]: that is specifically the compressed thing, which is not applicable here

Comment (Atlante45) [reply libraries/fbx/src/FBXWriter.cpp]: Gotcha

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3372/HighFidelity-Beta-PR11369-2a4f7712c32988ada4b5862f442199fefd837452.exe

**Pull request merged**